### PR TITLE
Update the OpenIdConnect Azure Sample

### PR DESCRIPTION
**Pull request by troydai**
Add a set up instruction.

### CODE REVIEW

File: samples/OpenIdConnect.AzureAdSample/Readme.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# How to set up the sample
+
+## Pre-requisit
+1. Set up an Azure Active Directory, save the AD's "Tenent name"
+2. Set up an Azure Web Site
```
Comment (Tratcher) [inline samples/OpenIdConnect.AzureAdSample/Readme.md]: Were you able to run the sample locally? You should be able to if you set the right UserSecrets.

Comment (troydai) [inline samples/OpenIdConnect.AzureAdSample/Readme.md]: Works for local, too.

File: samples/OpenIdConnect.AzureAdSample/Readme.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# How to set up the sample locally
+
+## Set up [Azure Active Directory](https://azure.microsoft.com/en-us/documentation/services/active-directory/)
+
+1. Create your own Azure Active Directory (AD). Save the "tenent name".
+2. Add a new Application: in the Azure AD portal, select Application, and click Add in the drawer.
+3. Set the sign-on url to `http://localhost:5000`.
```
Comment (Tratcher) [inline samples/OpenIdConnect.AzureAdSample/Readme.md]: The launch settings are set to http://localhost:42023

File: samples/OpenIdConnect.AzureAdSample/Readme.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+4. Select the newly created Application, navigate to the Configure tab.
+5. Find and save the "Client Id"
+8. In the keys section add a new key. A key value will be generated. Save the value as "Client Secret"
+
+## Configure the local environment
+1. Set environment ASPNETCORE_ENVIRONMENT to DEVELOPMENMT. ([Working with Multiple Environments](https://docs.asp.net/en/latest/fundamentals/environments.html))
```
Comment (Tratcher) [inline samples/OpenIdConnect.AzureAdSample/Readme.md]: Also already set in the launch settings. Are you running these in VS, VSCode, or commandline?

File: samples/OpenIdConnect.AzureAdSample/Readme.md (hunk 2 above)
Comment (troydai) [inline samples/OpenIdConnect.AzureAdSample/Readme.md]: The setting is in launchSettings.json, so VS will adopt it. However if you run the application from terminal using dotnet run, the port is 5000 instead.

File: samples/OpenIdConnect.AzureAdSample/Readme.md (hunk 3 above)
Comment (troydai) [inline samples/OpenIdConnect.AzureAdSample/Readme.md]: In terminal (command line).

Comment (Tratcher) [inline samples/OpenIdConnect.AzureAdSample/Readme.md]: "Set the environment variable ASPNETCORE_ENVIRONMENT..."

**Comment by Tratcher** [comment]
:shipit:

**Comment by troydai** [comment]
Rebase and squash. Waiting for CI.

**Pull request merged**